src/services/postService.js

```diff
@@ -4,8 +4,12 @@ const UserModel    = db.sequelize.models.user;
 const Op           = db.Sequelize.Op;
 const _            = require('lodash');
 const ReplyService = require('./replyService');
+const client       = require('./steem');
 const {
```

Comment:
Are we using steem client now? 
I think they offer a way to fetch posts in html format via their api. They have an example in their docs if I recall correctly. Might not even have to use Remarkable

Reply:
This has been addressed with Remarkable.